<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="Chrono" name="lcdNumber">
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>8</number>
       </property>
       <property name="value" stdset="0">
        <double>0.000000000000000</double>
       </property>
       <property name="intValue" stdset="0">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>801</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="startStopButon">
         <property name="text">
          <string>DÃ©marrer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="retenirButton">
         <property name="text">
          <string>Retenir</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>801</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>Exporter au format CSV</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Chrono</class>
   <extends>QLCDNumber</extends>
   <header>chrono.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
